src/components/loader-bar/loader-bar-test.stories.tsx

```diff
@@ -23,3 +23,7 @@ export default {
 export const Default = ({ size, ...args }: LoaderBarProps) => {
   return <LoaderBar size={size} {...args} />;
 };
+
+export const LoaderBarComponentTest = ({ ...props }) => {
```

Comment:
could we as well include only `Default` story to be shown in storybook list `Loader -> Test -> default`